src-java/kilda-pce/src/main/java/org/openkilda/pce/finder/FailReason.java

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2022 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.pce.finder;
+
+public enum FailReason {
+    NO_CONNECTION,
+    ALLOWED_DEPTH_EXCEEDED,
+    LATENCY_LIMIT,
+    MAX_WEIGHT_EXCEEDED,
+    HARD_DIVERSITY_PENALTIES
```

Comment:
Maybe we can add some string description for all values to have some human readable reason? 
for example like in enum `org.openkilda.messaging.Destination` :
```
HARD_DIVERSITY_PENALTIES("There is no non-overlapped protected path")
```

Reply:
Done